src/services/scanners/ScannerTwigFunction.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+namespace dmstr\lajax\translatemanager\services\scanners;
+
+use lajax\translatemanager\services\scanners\ScannerPhpFunction;
+use Yii;
+use yii\base\InvalidConfigException;
+use yii\helpers\Console;
+use yii\helpers\FileHelper;
+
+/**
+ * Class for processing twig files.
+ *
+ * Language elements detected in twig files:
+ * "t" functions:
+ *
+ * ~~~
+ * ::t('category of language element', 'language element');
```

Comment:
Are the examples correct?

Reply:
No, the colons should be removed @H-Algabri

Reply:
sorry, it's updated